dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -2991,6 +2994,58 @@
           }
         }
       }
+    },
+    "privateMedicalRecordAttachments": {
```

Comment:
I think it would be safer to use the attachments definition as the base for this property so that the definitions don't unintentionally diverge down the road. This would mean:
1. Move `attachments` schema to definitions
2. Set the `attachments` property to `{ $ref: '#/definitions/attachments' }
3. Set the `privateMedicalRecordAttachments` property to something like:
```
privateMedicalRecordAttachments: _.merge(
    { $ref: '#/definitions/attachments' },
    {
      items: {
        properties: {
          attachmentId: {
            enum: // whatever you need
            enumNames: // whatever you need
          }
        }
      }
    }
  )